package.json

```diff
@@ -13,6 +13,7 @@
   },
   "scripts": {
     "build": "tsup",
+    "prepare": "tsup",
```

Comment:
Since there is already the `build` script do you mind removing this again? or what was the reason to add and `prepare` script?

Reply:
I found it useful if someone try to fork this and install it directly from github. If you think it's not needed, I can remove it 😅

Example:
```
npm i github:rinn7e/kysely-pglite-dialect#rinn7e/pglite-0.3
```

Reply:
All good, looks actually really useful, thanks!

Reply:
Thank you too for creating this dialect 😃